test/fixtures.js

```diff
@@ -2582,7 +2589,158 @@ module.exports = function() {
             }
         },
 
-        // 77: optout cb245673-aa41-4302-ac47-00000000001 (sms_pmtct)
+        // 77: get identity by msisdn +27821234444 (no nc_working_on)
+        {
+            "request": {
+                "url": 'http://is.localhost:8001/api/v1/identities/search/',
+                "method": 'GET',
+                "params": {
+                    "details__addresses__msisdn": '+27821234444'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [{
+                        "url": "http://is.localhost:8001/api/v1/identities/cb245673-aa41-4302-ac47-00000034444/",
+                        "id": "cb245673-aa41-4302-ac47-00000034444",
+                        "version": 1,
+                        "details": {
+                            "default_addr_type": "msisdn",
+                            "addresses": {
+                                "msisdn": {
+                                    "+27821234444": {}
+                                }
+                            },
+                        },
+                        "created_at": "2016-08-05T06:13:29.693272Z",
+                        "updated_at": "2016-08-05T06:13:29.693298Z"
+                    }]
+                }
+            }
+        },
+
+        // 78: get identity by msisdn +27821231111 (user with working_on extra)
+        {
+            "request": {
+                "url": 'http://is.localhost:8001/api/v1/identities/search/',
+                "method": 'GET',
+                "params": {
+                    "details__addresses__msisdn": '+27821231111'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [{
+                        "url": "http://is.localhost:8001/api/v1/identities/cb245673-aa41-4302-ac47-00000031111/",
+                        "id": "cb245673-aa41-4302-ac47-00000031111",
+                        "version": 1,
+                        "details": {
+                            "default_addr_type": "msisdn",
+                            "addresses": {
+                                "msisdn": {
+                                    "+27821231111": {}
+                                }
+                            },
+                        },
+                        "nc_working_on": "+27821232222",
```

Comment:
Put this inside `details`, inside a sub-object called `nurseconnect`, and call it `working_on`, since the `nc_` would then be redundant.